### PR TITLE
updated php sdk to sanitize card holder details

### PR DESCRIPTION
**Pull request by SenthilKumar-86**

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 === Events Manager Pro SecureSubmit Gateway ===
 Contributors: markhagan
 Tags: events, event, event registration, event calendar, events calendar, event management, addon, extension, addition, registration, ticket, tickets, ticketing, tickets, theme, widget, locations, maps, booking, attendance, attendee, calendar, gigs, payment, payments
-Tested up to: 3.9.1
+Tested up to: 4.7
```
Comment (ttnnkkrr) [inline README.txt]: @SenthilKumar-86 Any chance we can put the latest version of PHP you tested with?

File: class-events-manager-securesubmit.php [hunk 2]
```diff
@@ -137,7 +138,7 @@ public function __construct() {
 
             add_filter( 'em_wp_localize_script', array( $this, 'em_wp_localize_script' ), 10, 1 );
             add_filter( 'em_booking_form_action_url', array( $this, 'force_ssl' ), 10, 1 );
-            wp_enqueue_script('hps_em_securesubmit_library', plugins_url('jquery.securesubmit.js',__FILE__), array('jquery'));
+            wp_enqueue_script('securesubmit', plugins_url('jquery.securesubmit.js',__FILE__), array('jquery'));
```
Comment (slogsdon) [inline class-events-manager-securesubmit.php]: @SenthilKumar-86 I'd like to keep this as it was. The change to `hps_em_securesubmit_library` was originally made to combat conflicts with our other plugins that have already upgraded to our pure JavaScript `securesubmit.js`

Comment (SenthilKumar-86) [inline class-events-manager-securesubmit.php]: The js handle is updated as hps_em_securesubmit_library

File: README.txt (hunk 1 above)
Comment (SenthilKumar-86) [inline README.txt]: PHP version updated in the readme file

**Pull request merged**